### DISCUSSION TITLE
View from Neptune

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3259] [Show and tell]
I took a picture of the Solar System December 2022. Stellarium was set on Neptune. My friend was surprised. The Sun was dimmer than Jupiter. Is this realistic? Here is picture on this day 3th June 2023.  Finnish translation: Sun (Aurinko), Earth (Maa).
![image](https://github.com/Stellarium/stellarium/assets/18149564/d392d06b-629c-4751-a3ff-13dbc0c12b72)

**Comment by 10110111** [comment]
No it's not realistic. Recall the Pale Blue Dot shot by Voyager 1 from 40.5 AU away, where the glare from the Sun almost masks the Earth. Jupiter would be a brighter dot, but still incomparable to the Sun itself.

**Comment by Atque** [comment]
It is worth noting that the Mallama magnitude formula for Jupiter is only valid for phase angles < 130°.

**Reply by gzotti** [reply]
Yes. It's easy (and correct) to state that in a paper. But how to fill the other area in a visualisation? Optimizing "here" may break "there". Stellarium is by default optimized for earth views. But the issue is not about Jupiter shown too bright. Obviously, the sun is brighter than anything else in the planet area. It appears (without check) that the sun is clipped away by distance, leaving just the "halo" (glow). Maybe somebody has optimized away an extra glow because somebody else has complained it should not be there, without having this use case in mind.
